tests/suites/main_test.function

```diff
@@ -140,14 +140,19 @@ int get_line( FILE *f, char *buf, size_t len )
 {
     char *ret;
 
-    ret = fgets( buf, len, f );
-    if( ret == NULL )
-        return( -1 );
+    buf[0] = '#';
 
-    if( strlen( buf ) && buf[strlen(buf) - 1] == '\n' )
-        buf[strlen(buf) - 1] = '\0';
-    if( strlen( buf ) && buf[strlen(buf) - 1] == '\r' )
-        buf[strlen(buf) - 1] = '\0';
+    while( buf[0] == '#' )
```

Comment:
Wouldn't a do while loop be more descriptive of itself? Looking straight on this line in github diff makes me think something is wrong. But a whole view helps.

Reply:
@mazimkhan You are perfectly right about do...while, thank you. Fixed. Also I moved the line break stripping out of the loop.